osbs/conf.py

```diff
@@ -510,3 +511,24 @@ def get_low_priority_node_selector(self):
             nodeselector = dict([k.strip(), v.strip()] for (k, v) in raw_nodeselector.items())
 
         return nodeselector
+
+    def get_equal_labels(self):
+        equal_labels = []
+        equal_labels_str = self._get_value("equal_labels", self.conf_section,
+                                           "equal_labels")
+        if equal_labels_str:
+            # must be in correct format
```

Comment:
I know its too late, but why we didn't chose json for this? dict of lists for equal preference labels would do the trick, I guess